packages/test/src/components/swap/TokenSwap.tsx

```
@@ -0,0 +1,675 @@
+    }
+  };
+
+  // Accessing global swap state using Zustand store
+  const {
+    swapParams,
+    inputAmount,
+    btcAddress,
+    errorMessage,
+    loading,
+    isBtcToWbtc,
+    setSwapParams,
```

ESLint

[error] 'setSwapParams' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)